GameServerLib/Logic/Players/PlayerManager.cs

```diff
@@ -28,7 +30,7 @@ public void AddPlayer(KeyValuePair<string, PlayerConfig> p)
             };
```

Comment:
Yeah, somewhere here just parse `var TeamId = GetTeamIdFromSomewhere();` and use that later on in the code. We want all teams to use the same code, and no separate if/else clauses for them like there currently is.